### ISSUE TITLE
How to train on custom dataset with ground truth masks?

### ISSUE THREAD

**Issue by Irennnne** [wusize/CLIPSelf#22]
I have my own dataset which contains the ground truth semantic segmentation masks for each image. How can I prepare the embeddings and region proposals for fine-tuning? Can I directly use the gt bboxes as the region proposals? Thank you!
